src/app/partials/threshold.scss

```diff
@@ -0,0 +1,28 @@
+
+$tablet-threshold: 900px;
+$mobile-threshold: 750px;
+
+@mixin smaller($devicetype) {
+  @if $devicetype==mobile {
+    @media screen and (max-width: $mobile-threshold) {
+      @content;
+    }
+  }
+  @else if $devicetype==tablet {
+    @media screen and (max-width: $tablet-threshold) {
+      @content;
+    }
+  }
+}
+
+@mixin bigger($devicetype) {
+  @if $devicetype==mobile {
+    @media screen and (min-width: $mobile-threshold) {
+      @content;
+    }
+  } @else if $devicetype==tablet {
+    @media screen and (min-width: $tablet-threshold) {
+      @content;
+    }
+  }
```

Comment:
it will be more readable if we call these mixin's as `smallerThan` and `biggerThan` as they function like that. Also `deviceType` doesn't fit with the intent here we may call that `deviceBreakpoint` as it's value doesn't define a device but a point form where one side is let's say mobile and other is tablet. And they will be used like `screen.smallerThan(mobileBreakpoint)` and `screen.biggerThan(tabletBreakpoint)` which sounds more descriptive than current one.